### ISSUE TITLE
I cannot scrape music Artists

### ISSUE THREAD

**Issue by Santicoma** [Komet/MediaElch#1681]
**Scrapers that don't work:**

 - Universal Music Scraper

**Details which are wrong / not loaded:**
Add a list of information that is not loaded correctly.
The scraper page open with the Artist Name, but after clicking It doesn't save any information, doesn't start

**MediaElch Version:**
 - [ ] 2.10.5-dev (nightly)
 - [ ] 2.10.4 (stable)

**Operating System:**
 - [ ] Windows

**Additional context:**
Until one month ago I used mediaElch to scrape music, but then I began to scrape Concerts and TV Shows, and left scrape music.

**Comment by bugwelle** [comment]
Hi,

thanks for reporting. Do you use the latest _stable_ version or the latest nightly version? :)

I've fixed a few issues with the universal music scraper. I'll release a new version shortly.

Regards,
Andre

**Comment by bugwelle** [comment]
v2.10.6 is released. Could you test it with the latest version? :)

**Comment by Boulder08** [comment]
The same happens with 2.10.6. No matter what the scraper is, no results are displayed.

**Comment by bugwelle** [comment]
Hi,

I still can't reproduce the issue. Could you tell me:

1. Which scraper is preferred in MediaElch's scraper settings? (i.e. the preferred scraper for the universal music scraper)
2. Which checkboxes you select in the scrape dialog
3. If you scrape a single artist or multiple ones
4. For which artists this happens?

Regards,
Andre

**Comment by Boulder08** [comment]
1) I've tried all the scrapers there, same behaviour. The search field shows for a brief moment (like a fraction of a second) that it's searching, then no results.
2) I've tried all, and Name only.
3) No change, multiple artists seems to go through the selected ones like it does for a single artist, very short searching time for each.
4) For example Anathema, Black Sabbath.

To me it looks like the Universal Music Scraper itself is giving grief. Is there a way to debug what it actually does and gets as a result? I'd be happy to help you with that information.

**Comment by bugwelle** [comment]
Thank you. Are there at least search results in the scrape dialog?

Sorry about the issue. It's often easy to fix once I've reproduced the issue. But for me, searching works. I will have to test it on my Windows virtual machine...
Do you use Windows 10 or 11 or another OS?

![Screenshot_20240127_174614](https://github.com/Komet/MediaElch/assets/1667306/1fb0e4e9-d08d-4f80-b514-97b74ff941cd)

**Comment by Boulder08** [comment]
No, the dialog is completely empty which is why I suspected that it's due to the scraper itself.

![image](https://github.com/Komet/MediaElch/assets/11799308/3708a8e6-44c8-4a76-9b32-6e94df77f6c6)

I'm using Win10 with the latest updates.

**Comment by bugwelle** [comment]
Tested on Windows 10 with v2.10.6 and it works. hmm...

What language did you select for Universal Music Scraper in MediaElch's settings?

Also, could you provide a debug log if possible? You can enable it via an `advancedsettings.xml` file next to `MediaElch.exe`.
See https://mediaelch.github.io/mediaelch-doc/settings.html#advanced-settings
Copy this file and change `<debug>false</debug>` to `<debug>true</debug>`: https://github.com/Komet/MediaElch/blob/9f8892fb919fbb857b7696d2c16b356d416f379c/docs/advancedsettings.xml#L24-L32

![Screenshot_20240128_123838](https://github.com/Komet/MediaElch/assets/1667306/eade45e7-2b88-4427-9f8a-748dd7946974)

**Comment by bugwelle** [comment]
Also, maybe as a last resort: If you go and open https://musicbrainz.org/ws/2/artist/?query=artist:"Rainbow" , do you see anything? It should be an XML file (`<metadata created…`).

**Comment by Boulder08** [comment]
With the help of the debug log, I was able to find out the reason! There was a network error and connection was just closed with any attempt. As I'd seen problems with various sites, I tested _**disabling my VPN connection**_ and the search started working right away. There were also some connection issues with TMDb in the log but they disappeared when connecting without VPN.

Can you list the sites that are being connected with ME? I should put them all in the VPN bypass list to avoid any further issues.
